ToDoList-backend/ToDoList.Api/Assignees/Models/AssigneeDTO.cs

```diff
@@ -1,11 +1,11 @@
 ﻿using System.Collections.Generic;
-using ToDoList.Api.BucketTasks.Models;
+using ToDoList.Domain.Models;
 
 namespace ToDoList.Api.Assignees.Models;
 
 public record AssigneeDto
 {
     public int Id { get; set; }
     public string Name { get; set; } = string.Empty;
-    public List<BucketTaskDto> BucketTask { get; set; } = new List<BucketTaskDto>();
```

Comment:
Dobrze bylo z tym Dto, dlaczego zmienilismy, dto nie powinno osadzac w sobie modeli bazodanowych, jedynie inne dto :/ Do tego lepiej bylo moim zdaniem zainicjalizowac te wlasciwosc pusta lista niz dawac `?`. Ponawiam propozycje wprowadzenia rekordow `public record AssigneeDto(int Id, string Name, List<BucketTaskDto> BucketTasks)`. Pamiętajmy też, że jeśli właściwośc jest kolekcją, np. listą, to dobrze jest użyć liczby mnogiej, czyli `BucketTasks` a nie `BucketTask`.

Widzę, że analogiczna zmiana była w innych miejscach też.

Reply:
Fajnie Rafał, dzięki za wychwycenie tego bubla. Zrobiłem tak z rozpędu, ponieważ do obsługi pola formularza z assignee search potrzebowałem AssigneeId ale to faktycznie wystarczy tylko w dto od bucket tasków żeby działało.
Zatem przerobiłem dtosy na rekordy usuwając przy tym właściwości nawigacyjne Entity Framework'a :)